cypress/integration/hakemuseditoriSpec.ts

```diff
@@ -0,0 +1,115 @@
+import * as hakemuseditori from '../hakemuseditori'
+import * as hakemuksentaytto from '../hakemuksentaytto'
+import * as routes from '../routes'
+import { unsafeFoldOption } from '../option'
+
```

Comment:
Löytyisivätkö testit helpommin alihakemistosta nimeltä 'test' tai 'spec'?

Reply:
Hyvä kysymys. Cypressin omassa dokumentaatiossa suositellaan nimenomaan tuota rakennetta. Ajattelin, että selkeyden vuoksi myös me käytettäisiin kyseistä rakennetta, jotta meidän käytännöt olisivat yhtenevät ns. "yleisten suositusten" kanssa.

https://docs.cypress.io/guides/core-concepts/writing-and-organizing-tests.html#Folder-Structure